src/chapters/html/semantic-html.rst

```diff
@@ -0,0 +1,335 @@
+
+For example, developers can make paragraph text larger than heading text if
+they want to. By looking at the HTML, however, we can still identify what is
+the paragraph and what is the heading. The ``<p>`` and ``<h>`` tags give us
+that information. This is the strength of semantic HTML. The tag names express
+meaning.
+
+Another benefit to semantic HTML is that programmers find it easier to
+visualize the results when they can put a specific name to a section of their
+code.
+
+Screen Readers and Accessibility
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A well designed webpage organizes content in a clean and attractive way. For
+these pages, sighted users can easily find the navigation bar or identify the
+main content of the page. However, individuals with visual impairments may use
+screen readers to navigate the page.
+
+Screen readers are tools that verbally describe the content on a page. They
+also describe the HTML tags surrounding the content. With semantic HTML, we can
+make our webpages accessible to all users.
+
+For example, the ``div`` and ``span`` elements organize content on the page,
+but they do not provide any information about the content itself. If we use
+separate ``div`` elements to build a navigation bar and organize a set of blog
+posts, a screen reader can give no clarification to the user. As the person
+listens, they might not recognize which words deal with navigation. However, if
+we replace a ``<div>`` tag with ``<nav>``, the device can alert the user that
+the elements inside provide links to key locations.
+
+A screen reader just reads what's in each element. With semantic HTML, the
+device can inform the user that they have reached a navigation bar, a header, a
+login form, a news article, etc. This helps the user understand the layout of
+the page.
+
+Without semantic HTML, programmers must add extra attributes (like
+``class="nav-bar"``) to assist those who rely on screen readers.
+
+Search Engines
+^^^^^^^^^^^^^^
+
+Search engines also benefit from semantic HTML. If we can communicate which
+part of our page holds the most important content, we give the software a
+better chance to find our website and show users an accurate description.
+
+For example, assume we create a website with the title "My Great Site", and we
+follow that title with with a collection of ``div`` elements. When a user
+types *My Great Site* into a search engine, the software will find our site and
+display a description of the content. Since ``div`` tags provide no information
+about the content they contain, the description of our site may be incomplete.
+
+Carefully using semantic HTML tags helps search engines find and steer users
+toward our home page. If we include a ``main`` element with some ``article``
+elements inside, the engines find our most important content faster and provide
+a more accurate description.
+
+With search, speed an accuracy are critical! Semantic tags let us tell search
```

Comment:
"With search engines, speed and accuracy..."

Reply:
Fixed.